tests/integration/reusable_values.py

```diff
@@ -95,6 +98,26 @@ async def _set_up_reusable_values():
     amm_asset2 = setup_amm_pool_res["asset2"]
     amm_issuer_wallet = setup_amm_pool_res["issuer_wallet"]
 
+    # Create and accept one credential; This is used in the PermissionedDomain tests
```

Comment:
This is not necessary. You can create a Permissioned Domain without any credentials existing for it.

Reply:
that is true, but I'd like to test for a `tesSUCCESS` code at the end. If the credential-object doesn't exist, I wasn't sure if rippled throws an error.

Besides, this displays the general workflow with PD

Reply:
Nonetheless, a re-usable credential in the integration test cases would be useful for upcoming amendments. I prefer to retain this piece of code.